client/src/components/reviews/breakdownComponents/Breakdown.jsx

```diff
@@ -14,35 +14,34 @@ let ratingsCountTotal = (ratingsObj) => {
 }
 
 let averageRating = (ratingsObj, total) => {
-  if (total === 0) {
-    return 0;
-  }
+  // if there are no reviews you can't divide by zero so return 0;
+  if (total === 0) { return 0; }
```

Comment:
Is that the total number of votes or the total adding together of the votes values? That seems a little vague.

Reply:
It was the total that is passed in as a parameter. I can go back and change the name.